test/hotspot/jtreg/vmTestbase/nsk/jdb/hidden_class/hc001/hc001.java

```diff
@@ -318,18 +317,16 @@ private void testInvEvalCommand(String hcName) {
 
     /* Test the jdb commands "watch" and "eval" with various invalid class names. */
     private void testInvalidCommands() {
-        String className = null;
         String[] invClassNames = {
-            "xx.yyy/0x111/0x222",
-            "xx./0x111.0x222",
-            "xx.yyy.zzz/"
+                "xx.yyy/0x111/0x222",
+                "xx./0x111.0x222",
+                "xx.yyy.zzz/"
```

Comment:
Why are these indented 8 instead of 4?

Reply:
due to the same reasons in the case w/ `fields001`, these lines have 3 unit indentation, 1st for `hc001` class, 2nd for `testInvalidCommands` method, 3rd for `invClassNames` array initialization, so they have 3x4 = 12 spaces.

Reply:
File test/hotspot/jtreg/vmTestbase/nsk/jdb/kill/kill001/kill001.java has this change:

```
         for (int i = 0; i < threads.length; i++) {
             reply = jdb.receiveReplyForWithMessageWait(JdbCommand.kill + threads[i] + " " +
-                                                       DEBUGGEE_EXCEPTIONS + "[" + i + "]",
-                                                       "killed");
+                            DEBUGGEE_EXCEPTIONS + "[" + i + "]",
+                    "killed");
         }
```
I think, the second line "killed");  has to be aligned with the previous one.

Reply:
Hi Igor,

Overall, it is great. Your formatting tool seems to be AI. 👍 
This update fixes a lot of formatting issues.
I have no more comments so far.

Reply:
Regarding the "killed" formatting, I think the reason for it is the 8 space "line continuation rule". The 2nd line is a actually a line continuation of a line continuation, so it gets indented and extra 16 spaces. The 3rd line is just a single line continuation, so gets indented just an extra 8. I think what would help to clean this up is to move the first argument onto a newline, which would just be indented an extra 8. That way it won't be broken up over multiple lines, and will also be inline with the "killed" argument.

For the 2nd case above, I also think this is less readable than the original. I personally like it if you align all arguments with the first one, even if it leads to a non-standard indentation. If you want subsequent arguments to use the 8 space line continuation rule, then I suggest whenever arguments are on multiple lines that you always place the first argument on a new line so all arguments are aligned together.